src/gamecontrol.cpp

```diff
@@ -34,6 +35,7 @@ void OnNewYear()
 /** Runs various procedures that have to be done monthly. */
 void OnNewMonth()
 {
+	Autosave();
```

Comment:
If I load the save again, will it perform the other `OnNewMonth` updates?

Reply:
The save isn't performed immediately – `Autosave()` delegates to `SaveGame()` which only sets the game controller's `next_action` variable to `GCA_SAVE_GAME`, which is then carried out on the next frame. So all the monthly updates will be performed before the save, and the saved game will start one tick into the new month.